app/models.py

```diff
@@ -2218,3 +2218,26 @@ def serialize(self):
             'created_at': self.created_at.strftime(DATETIME_FORMAT),
             'updated_at': self.updated_at.strftime(DATETIME_FORMAT) if self.updated_at else None
         }
+
+
+class NotificationFailures(db.Model):
+    """
+    A SQLAlchemy model representing the 'notification_failures' table.
+
+    Attributes:
+        notification_id (int): The primary key of the table.
+        created_at (DateTime): Record date and time of creation. This field will be used for pg_cron
+        job delete records older then 30 days
+        body (JSONB): Column used to store the details of the notification failure in a JSON object.
+
+    Methods:
+        serialize: Converts the 'body' attribute to a dictionary for easier consumption in Python.
+    """
+    __tablename__ = 'notification_failures'
+
+    notification_id = db.Column(db.Integer, primary_key=True)
```

Comment:
Not understanding how this is an integer when a notification_id is always a UUID? Is that how the notifications table is configured?

Reply:
done